src/test/java/com/razorpay/AccountClientTest.java

```diff
@@ -230,4 +231,52 @@ private String getResponse(String account_id) {
                 "  }\n" +
                 "}";
     }
+
+    @Test
+    public void uploadAccountDoc() throws RazorpayException {
+        JSONObject request = new JSONObject();
+        request.put("files","/Users/your_name/Downloads/sample_uploaded.pdf");
```

Comment:
it is better to use relative URL instead of absolute URL

Reply:
https://github.com/razorpay/razorpay-java/blob/4a46d687d24522058165127a19cd9ac96b85fdd1/documents/account.md?plain=1#L410

Reply:
https://github.com/razorpay/razorpay-java/blob/4a46d687d24522058165127a19cd9ac96b85fdd1/src/test/java/com/razorpay/AccountClientTest.java#L238